regionService/srv/www/regionService/regionInfo.py

```diff
@@ -132,17 +130,43 @@ def create_smt_region_map(conf):
             region_smt_cert_fingerprints
         )
         region_name_to_smt_data_map[section] = smtInfo
-        for ipRange in region_public_ip_ranges.split(','):
+        for ip_range in region_public_ip_ranges.split(','):
             try:
-                net = ipaddress.ip_network(ipRange)
+                ipaddress.ip_network(ip_range)
             except ValueError:
                 msg = 'Could not proces range, improper format: %s'
-                logging.error(msg % ipRange)
+                logging.error(msg % ip_range)
                 continue
-            for host in net.hosts():
-                ip_to_smt_data_map[str(host)] = smtInfo
+            ip_range_to_smt_data_map[ip_range] = smtInfo
 
-    return ip_to_smt_data_map, region_name_to_smt_data_map
+    return ip_range_to_smt_data_map, region_name_to_smt_data_map
+
+
+# ============================================================================
+def find_longest_prefix_ipv4(requester, ip_range_map):
+    """Find IP ranges for IPv4 adresses that are likely to contain the
+       requester address"""
+    potential_ranges = {
+        1: [],
+        2: [],
+        3: []
+    }
+    results_map = {}
+    requester_prts = requester.split('.')
+    for ip_range in ip_range_map.keys():
+        ip_range_prts = ip_range.split('.')
```

Comment:
Same here - why abbreviate "parts" ?

Reply:
So my finger skin doesn't wear off, fixing this, it's habbitt